modules/host.go

```diff
@@ -92,6 +92,12 @@ type (
 		ObligationStatus    uint64 `json:"obligationstatus"`
 	}
 
+	// HostWorkingState reports the working state of a host.
+	HostWorkingState string
+
+	// HostConnectabilityState reports the connectability state of a host.
+	HostConnectabilityState string
```

Comment:
Hmm. In other modules I think we actually defined consts for each possible result. e.g. `statusChecking HostConnectabilityStatus = "checking"`

And, I'm thinking 'Status' is a better suffix than 'State', @lukechampine thoughts on that?